### ISSUE TITLE
Child theme support

### ISSUE THREAD

**Issue by timalbert** [roots/sage#1894]
## Submit a feature request or bug report

- [X] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/guidelines/blob/master/CONTRIBUTING.md)
- [x] This is a feature request
- [] This is a bug report
- [X] This request isn't a duplicate of an [existing issue](https://github.com/roots/sage/issues)
- [X] I've read the [docs](https://roots.io/sage/docs) and [NPM Debugging Guidelines post](https://discourse.roots.io/t/npm-debugging-guidelines-failed-npm-install-bower-install-or-gulp-build-read-this/3060) and followed them (if applicable)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/c/sage) forums

## Feature Request

**Please provide use cases for changing the current behavior:**

This is a request for child theme support for Sage 9. In almost every other way (templating, configuration, etc.) Roots does an amazing job of keeping WP structure and functionality intact. Implementing a parent/child theme however seems to be missing. By creating a child theme, you further abstract/extend functionality and style.

**Comment by raymondkingston** [comment]
Yes, please add this functionality! My team is effectively making 'department' websites, which are child themes using Sage as a parent. Sage 9 presently requires a separate instance for each department.

**Comment by mediaformat** [comment]
I was flabbergasted by this bug, and reassured upon seeing this issue!

**Comment by QWp6t** [comment]
Child themes should be supported once this  is merged: https://github.com/roots/sage/pull/1919/commits/1a8dce9b092e8bbe1d1921c43eb3691f3399b0a7

**Comment by andrewklau** [comment]
@QWp6t could you give some pointers on how to create one now that https://github.com/roots/sage/commit/1a8dce9b092e8bbe1d1921c43eb3691f3399b0a7 has merged?

**Comment by LukeAbell** [comment]
@andrewklau  @QWp6t did you ever get this figured out?

**Comment by soham2008xyz** [comment]
@QWp6t this is still an issue in Sage 9. Any word on child theme support?

**Comment by jvkrakowski** [comment]
Was this implemented at any point for Sage 9?

**Comment by raymondkingston** [comment]
@jvkrakowski I don't believe so. My team is on to Sage 10 at this point, and it still doesn't support child themes, which is a real shame for huge institutions with hundreds of sites that need their own levels of customization. Are ya hearing us, @QWp6t and Roots? 😉
